#!/usr/bin/env python3
PACKAGE = "ydpibot_bringup"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp_x", double_t, 1, "x kp",    1, 0,   200)
gen.add("ki_x", double_t, 1, "x ki",    0, 0,   200)
gen.add("kd_x", double_t, 1, "x kd",    0, 0,   200)
gen.add("kp_w", double_t, 1, "w kp",    1, 0,   200)
gen.add("ki_w", double_t, 1, "w ki",    0, 0,   200)
gen.add("kd_w", double_t, 1, "w kd",    0, 0,   200)

exit(gen.generate(PACKAGE, "ydpibot_bringup", "pid"))